web/src/app/administration/workbasket/details/workbasket-details.component.ts

```diff
@@ -46,31 +46,31 @@ export class WorkbasketDetailsComponent implements OnInit, OnDestroy {
 
   ngOnInit() {
     this.workbasketSelectedSubscription = this.service.getSelectedWorkBasket().subscribe(workbasketIdSelected => {
-      this.workbasket = undefined;
+      delete this.workbasket;
       this.getWorkbasketInformation(workbasketIdSelected);
     });
 
     this.routeSubscription = this.route.params.subscribe(params => {
       let id = params['id'];
-      this.action = undefined;
+      delete this.action;
```

Comment:
what about a first if(!id) return ... 

would be easier on all other ifs.

Reply:
@BVier why did you resolve this?